packages/bappo-components/src/primitives/ActivityIndicator/ActivityIndicator.native.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import RN from 'react-native';
+
+import { ActivityIndicatorProps } from './types';
+
+export default function ActivityIndicator({
+  animating = true,
+  color = '#999',
+  size = 'small',
+  style,
+  testID,
+}: ActivityIndicatorProps) {
+  return (
+    <RN.ActivityIndicator {...{ color, animating, size, style, testID }} />
```

Comment:
Unfortunately Activity Indicator doesn't take `testID`.  We can just remove it.

Reply:
Thanks Stan, I've removed the `testID`.